front/src/common/authorization/components/Error403.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import NavBarSNCF from 'common/BootstrapSNCF/NavBarSNCF';
+
+const Error403 = () => {
+  const { t } = useTranslation(['errors', 'common/common']);
+  const navigate = useNavigate();
+  return (
+    <>
+      <NavBarSNCF appName="OSRD" />
+      <main className="mastcontainer mastcontainer-no-mastnav d-flex align-items-center justify-content-center vh-100">
+        <div className="p-3 text-center">
+          <h1>{t('errors:403')}</h1>
+          <button
+            className="btn btn-primary btn-sm px-2 ml-2"
+            type="button"
+            onClick={() => navigate(-1)}
```

Comment:
Should we offer a link to go back? This will go to the page that triggered the error, most likely instantly redirecting to the 403 page once again. Maybe we should just offer a link to go home instead?

Reply:
yes, good point !